pkgs/misc/vscode-extensions/update_installed_exts.sh

```diff
@@ -0,0 +1,65 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p curl jq coreutils
+set -eu -o pipefail
+
+function fail() {
+    echo $1 >&2
+    exit 1
+}
+
+function fetch() {
+    RES=$(curl --silent "$1")
```

Comment:
`--silent --show-error --fail` and fail on error is better then checking for `404` because it also catch other server errors (500) and network failure.

Reply:
I've removed that function and reworked `get_ver` to be more adept at handling errors.